### ISSUE TITLE
Dialogue improvements

### ISSUE THREAD

**Issue by cooljeanius** [cooljeanius/Son_Of_The_Black_Eye_Easy_Edition#3]
- [ ] better yeti-related dialogue in general
- [x] Mountains of Haag: make dwarves react to getting looted (requires coming up with an image for the responding villages)
- [ ] Civil War: ensure enemy leaders don't end up talking to themselves
- [x] Civil War: ensure Kapou'e doesn't think there are still enemy leaders after you've converted enemy sides to allies via assassination
- [ ] Civil War: reduce duplication in end-of-level dialogue
- [x] The Human Attack: additional text explaining that ally side is the one getting the gold when an allied unit kills a Royal Guard
- [x] General copyediting/proofreading, as done in mainline SotBE

**Comment by cooljeanius** [comment]
Point 4 ought to be done by 04c6b85

**Comment by cooljeanius** [comment]
So, for checkbox 2, in 6222f5b I came up with a different solution of creating Dwarvish Woodcutter units to react, instead of trying to find the image for the village in question

**Comment by cooljeanius** [comment]
For checkboxes 3, 4, and 5, here's another request for improving dialogue in that scenario: https://forums.wesnoth.org/viewtopic.php?p=667372#p667372
(yes, I already completed my point 4, but still, it's another data point towards the dialogue in that scenario needing improvement in general)
@brirush84 any thoughts?

**Comment by cooljeanius** [comment]
Putting off until post-1.16

**Comment by cooljeanius** [comment]
Also I'll want to copy over any of the relevant copyediting @max-torch did in wesnoth/wesnoth@c80f7f7 for wesnoth/wesnoth#6166 over to here, too (after reviewing it)

**Comment by max-torch** [comment]
Is it okay if I help out on this only after the copyediting I did gets copied? My concern is that I will be fixing/editing certain things I already previously fixed.

**Comment by cooljeanius** [comment]
> Is it okay if I help out on this only after the copyediting I did gets copied? My concern is that I will be fixing/editing certain things I already previously fixed.

ok yeah

**Comment by cooljeanius** [comment]
> Is it okay if I help out on this only after the copyediting I did gets copied? My concern is that I will be fixing/editing certain things I already previously fixed.

ok done in d81d165

**Comment by cooljeanius** [comment]
>     * [ ]  The Human Attack: additional text explaining that ally side is the one getting the gold when an allied unit kills a Royal Guard

I just realized I did this in e5ad90a without checking it off; doing so now

**Comment by cooljeanius** [comment]
@max-torch I think it's ready to take another look?

**Comment by cooljeanius** [comment]
Issue #23 could also fit under here, but instead I decided to make a separate issue for it
